cmd/policy/cpa_utils.go

```diff
@@ -0,0 +1,103 @@
+package policy
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	yaml "gopkg.in/yaml.v3"
+
+	"github.com/CircleCI-Public/circle-policy-agent/cpa"
+)
+
+//getPolicyDecisionLocally takes path of policy path/directory and input (eg build config) as string, and performs policy evaluation locally
+func getPolicyDecisionLocally(policyPath, input string) (*cpa.Decision, error) {
+	var config interface{}
+	if err := yaml.Unmarshal([]byte(input), &config); err != nil {
+		return nil, fmt.Errorf("invalid config: %w", err)
+	}
+
+	documentBundle, err := getDocumentBundleFromPath(policyPath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get document bundle for path: %w", err)
+	}
+
+	parsedPolicy, err := cpa.ParseBundle(documentBundle)
```

Comment:
Should use `cpa.ParsePolicy` so that linting errors can be caught during local development